gradle/project.gradle.kts

```diff
@@ -172,52 +175,26 @@ tasks {
         }
     }
 
-    val kotlinc by creating {
+    val kotlinc by creating(Download::class) {
```

Comment:
Is Download our class?

Reply:
No it's by a plugin, but it provides a great Download-Bar which means I was able to remove a lot of code